process/OpenSourceDevelopmentOpenHWGroup.md

```diff
@@ -0,0 +1,58 @@
+# OpenSource Development at OpenHW Group
+We encourage people to get involved and contribute to OpenHW Group projects.
+
+You can get involved in multiple ways:
+- File an issue (either against design artifacts, testbenches or the documentation - we consider errors in all of these to be bugs!).
+- Ask a question.
+- Generate a pull-request.
+- Propose a derivative project.
+
+As with any organization, at the OpenHW Group we have our ways-of-working that help keep entropy under control. Specifically, we use git, GitHub and the Eclipse development process.
+
+### Git/GitHub Development Workflow
+Here is a brief description of our development workflow:
+
+1. (Optional) Reach out and open an issue (or grab an open issue) for the feature you want to implement. The issue can be a great way of raising the attention and is also useful to track the implementation of certain features (which can go over multiple pull requests).
+
+2. Fork the repository into your own namespace using GitHub (you only need to do this once!). Configure the upstream remote.
+
+3. Synchronize the remote fork and bring it up to date:
+```
+$ git fetch --all
+$ git checkout master
```

Comment:
should that be main?

Reply:
Good catch.  Main is, in fact, the new name for the root branch of all GitHub hosted repos.  However, it _was_ master until a couple of years ago and we have a large set of repos that use master.  So we have three options:
1. Leave it as is (I do not like this one).
2. Change it to `$ git checkout main`
3. Change it to `$ git checkout main   # or "master" for older repos`

What do you think?

Reply:
how about 
4. Change it to $ git checkout main   # or "master" (depending on the name of the main branch for that repo)